### DISCUSSION TITLE
Start backup for all profiles

### DISCUSSION THREAD

**Discussion by dtu-compute** [borgbase/vorta#805] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have profiles for each type of data I want to backup. E.g. some I want each hour, others once per week and one only each month.

**Describe the solution you'd like**
Instead of I having to select each profile and then click Start backup, I would like a button "Start backup for all profiles".

**Comment by m3nu** [comment]
I could imagine running them one after another (serial). Parallel may not be a good idea due to high resource usage.

Another way is to simply use the *Schedule* tab to run them every hour/day/etc. That way you can time it somewhat.

Personally I have some backups on a daily schedule (to remote repos) and a few on manual (to USB/thumb drives). Running *all* of them at once would be a bit rough. Then people would ask to have a feature so they can pick which ones to include in *Run All*.

**Comment by Hofer-Julian** [comment]
I am not really convinced that another "Start backup for all profiles" is a good idea, mostly because of reasons stated by @m3nu and because we already don't have a lack of buttons.
